doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md

```diff
@@ -1,23 +1,23 @@
-## Procedural documentation for DSFinV-K generation
+# Procedural documentation for DSFinV-K generation
 
 
-### DSFinV-K export
+## DSFinV-K export
 
 You can download the current version of the DSFinV-K specification [here](https://www.bzst.de/DE/Unternehmen/Aussenpruefungen/DigitaleSchnittstelleFinV/digitaleschnittstellefinv_node.html).
 
-Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, shows how the previously described input values are mapped by fisklatrust to the files and data of the DSFinV-K export and defines how additional, for the DSFInV-K required, values can be sent to the ft.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge). 
+Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, how the previously described input values are mapped by fiskaltrust to the files and data of the DSFinV-K export, and defines how additional, for the DSFinV-K required values can be sent to the fiskaltrust.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge).  
```

Comment:
i do not understand the part that says: _, and defines how additional, for the DSFinV-K required values can be sent to the ..._ does it mean that there are extra values, that are additional?? so..
..and defines how values additional to the DSFinV-K required ones can be sent to..?

Reply:
i'll try rephrase it